src/api/app/controllers/webui/kiwi/repositories_controller.rb

```diff
@@ -1,12 +1,46 @@
 module Webui
   module Kiwi
     class RepositoriesController < Webui::WebuiController
+      before_action :set_image
+
       def index
-        @image = ::Kiwi::Image.find(params[:kiwi_image_id])
         @repositories = @image.repositories.order(:order)
         @package = @image.package
         @project = @package.project
       end
+
+      def bulk_edit
+        @repositories = @image.repositories.order(:order)
+        @package = @image.package
+        @project = @package.project
+      end
+
+      def bulk_update
```

Comment:
Can we just call these methods `edit` and `delete`? Even though it accepts nested attributes for multiple repositories, look at this rails casts example, he still follows the usual naming conventions:

https://github.com/ryanb/railscasts-episodes/blob/master/episode-197/surveysays/app/controllers/surveys_controller.rb

http://railscasts.com/episodes/197-nested-model-form-part-2